packages/editor/src/systems/EditorControlSystem.ts

```diff
@@ -362,8 +362,14 @@ const execute = () => {
       clickStartEntity = capturingEntity
     }
   }
+
   if (buttons.PrimaryClick?.up && !buttons.PrimaryClick?.dragging) {
-    if (hasComponent(clickStartEntity, SourceComponent) && !getState(ClickPlacementState).placementEntity) {
+    const urlParams = new URLSearchParams(window.location.search)
+    if (
+      hasComponent(clickStartEntity, SourceComponent) &&
+      !getState(ClickPlacementState).placementEntity &&
+      urlParams.get('wizardEnabled') === 'false'
```

Comment:
`wizardEnabled` comes from the irpro-wizard repo
is there a way we can dynamically do this?
(what if we try to disable it in the irpro-wizard itself by copying some required parts of the EditorControlSystems)